src/pf/anf.lua

```diff
@@ -247,7 +247,8 @@ local function renumber(expr)
    end
    local function lookup(var)
       if var == 'len' then return var end
-      return assert(substs[var], "unbound variable: "..var)
+      local subst = assert(substs[var], "unbound variable: "..var)
+      return subst
```

Comment:
I don't understand the change :)  How was the old code returning a different number of values?